### PR TITLE
refactor!: remove particle DATABASE global

### LINKED ISSUE #219: Get rid of global DATABASE instance

### PR DESCRIPTION
**Pull request by spflueger**
Closes #219

### CODE REVIEW

**Comment by spflueger** [comment]
@Leongrim @redeboer Be sure to check out the new `particles` notebook, if you like the changes in the user interface!

File: examples/particles.ipynb [hunk 1]
```diff
@@ -274,7 +272,10 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "particle.load_particles(\"additional_particles.yml\")"
+    "from expertsystem.io import load_particle_collection\n",
+    "\n",
+    "additional_particles = load_particle_collection(\"additional_particles.yml\")\n",
+    "particle_db.merge(additional_particles)"
```
Comment (redeboer) [inline examples/particles.ipynb:278]: You can also do:
```python
particle_db += load_particle_collection("additional_particles.yml")
```

File: tests/conftest.py [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import pytest
+
+from expertsystem import ui
+from expertsystem.data import ParticleCollection
+
+
+@pytest.fixture(scope="module")
```
Comment (redeboer) [inline tests/conftest.py:7]: Why not `"package"`?

**Review by redeboer — APPROVED** [review]
LGTM, just a few notes

File: tests/conftest.py (hunk 2 above)
Comment (redeboer) [reply tests/conftest.py:7]: Ah nvm, some of the tests modify this instance, right?

Comment (spflueger) [reply tests/conftest.py:7]: Yeah exactly

File: examples/particles.ipynb (hunk 1 above)
Comment (spflueger) [reply examples/particles.ipynb:278]: 851205813fc1b65715ce96956de4eba3af04c9cb

**Pull request merged**